commcare-cloud/commcare_cloud/argparse14.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import print_function
```

Comment:
A comment explaining this file would be good.

Reply:
👍 also cleaned this up a bit